journal/modernized/01-journal.adoc

```diff
@@ -135,13 +135,24 @@ named Morgan Evan, of South Wales:
 he had met with the people called Quakers in his travels, and was convinced of the truth.
 This poor man discoursed with my master about the principles of truth,
 and I being in the shop about my calling, my mistress came and said,
-Why do you not go out to help your master?`"for there
+Why do you not go out to help your master? for there
 is a Quaker at the door that has put him to silence.
 I hearing this, made haste, and took my bible under my arm,
 and put on what courage I could to dispute with that poor man,
 but he proved too hard for us all; when I went to them,
 they were upon the words Thee and Thou; but I very peremptorily asked him,
-what command he had to speak Thee and Thou; for I did acknowledge to him,
+what command he had to speak Thee and Thou;^
+footnote:[At this time in history, the correct and plain use of "`thee`" and "`thou`"
+to one person was beginning to give way to "`you`" and "`your.`"
+Most modern English speakers are unaware that the words "`you`" and "`your`" were originally plural pronouns used only to address two or more people,
+whereas "`thee`" and "`thou`" were used to address a single person. In the 1600's,
```

Comment:
In general, when you're adding stuff to an asciidoc, try to break on sentences.  So, let this line end with `person.` and then start the next line with `In the 1600's,...`